dev-node/anymatch/anymatch-3.1.1.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit node
+
+DESCRIPTION="Matches strings against configurable strings, globs, regular expressions, and/or functions"
+HOMEPAGE="
+	https://github.com/micromatch/anymatch
+	https://www.npmjs.com/package/anymatch
+"
+LICENSE="ISC"
+KEYWORDS="~amd64"
+NODEJS_RDEPEND="
+	dev-node/normalize-path
+	dev-node/picomatch
+"
```

Comment:
I don't think your script is appending a newline if dependencies are defined

Reply:
It's because I deleted code manually and sometimes the newline disappeared. I'll fix it later, now I'm writing stuff while ssh-ing from a smartphone